lib/cfnvpn/templates/lambdas/auto_route_populator/quotas.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+
+ROUTE_TABLE_QUOTA_CODE = 'L-401D78F7'
+AUTH_RULE_TABLE_QUOTA_CODE = 'L-9A1BC94B'
+EC2_SERVICE_CODE = 'ec2'
+IN_PROGRESS = ['PENDING', 'CASE_OPENED']
+
+def get_route_count(endpoint) -> int:
+  client = boto3.client('ec2')
+  response = client.describe_client_vpn_routes(
+    ClientVpnEndpointId=endpoint,
+  )
+  return len(response['Routes'])
+
+def quota_request_open(quota_code) -> bool:
+  client = boto3.client('service-quotas')
+  response = client.list_requested_service_quota_change_history_by_quota(
```

Comment:
Is it worth paginating this?

Reply:
probably not as the request if for the specific quota